matchminer/custom.py

```diff
@@ -1423,12 +1424,13 @@ def run_ctims_matchengine_job(trial_internal_ids):
             db_name='matchminer',
             ignore_run_log=True,
             ignore_report_date=True,
-            protocol_nos=trial_internal_ids
+            protocol_nos=trial_internal_ids,
+            failed_protocol_nos={}
```

Comment:
failed_protocol_nos isn't a configurable value, and it's defined in the engine itself, so no need to pass this information to the PMatchEngine

Reply:
ok